website/source/docs/agent/index.html.md

```diff
@@ -85,6 +87,19 @@ configuration entries:
   security of data transmissions to and from the Vault server. This value can
   be overridden by setting the `VAULT_SKIP_VERIFY` environment variable.
 
+### listener Stanza
+
+Agent supports one or more [listener][listener_main] stanzas.  In addition to
+the standard listener configuration, an Agent's listener configuration also
+supports an additional optional entry:
+
+- `require_request_header (bool: optional)` - Require that all incoming HTTP
+  requests on this listener must have an `X-Vault-Request: true` header entry.
+  Using this option offers an additional layer of protection from Service Side
+  Request Forgery attacks when the Agent has `auto-auth` turned on.  Requests on
```

Comment:
I don't think you need to call out auto-auth here since the two settings are independent of one another.